code-challenges/value-vs-reference/challenges-02.test.js

```diff
@@ -71,7 +68,7 @@ console.log(people[1].isAuthor) prints true
 ------------------------------------------------------------------------------------------------ */
 
 const setStatusAsAuthor = (people) => {
-  // Solution code here...
+  people.forEach(element => {element.isAuthor = true;});

```

Comment:
It would help make your code more readable to use meaningful parameter names inside of your forEach method. For instance, you could have used 'person' instead of 'element'